WhoIsThatServer.Storage/UnitTests/FaceFeaturesHelperTests.cs

```diff
@@ -0,0 +1,129 @@
+using System.Collections.Generic;
+using System.Linq;
+using FakeItEasy;
+using NUnit.Framework;
+using Shouldly;
+using WhoIsThatServer.Storage.Context;
+using WhoIsThatServer.Storage.ErrorMessages;
+using WhoIsThatServer.Storage.Exceptions;
+using WhoIsThatServer.Storage.Helpers;
+using WhoIsThatServer.Storage.Models;
+
+namespace WhoIsThatServer.Storage.UnitTests
+{
+    public class FaceFeaturesHelperTests
+    {
+        [Test]
+        public void InsertNewFeaturesModel_ShouldReturn()
+        {
+            //Arrange
+            var fakeFeaturesList = new List<FaceFeaturesModel>();
+            var fakeDbSet = UnitTestsUtil.SetupFakeDbSet(fakeFeaturesList.AsQueryable());
+
+            var fakeDbContext = A.Fake<DatabaseContext>();
+            A.CallTo(() => fakeDbContext.FaceFeatures).Returns(fakeDbSet);
+
+            var fakeDbContextGeneration = A.Fake<IDatabaseContextGeneration>();
+            A.CallTo(() => fakeDbContextGeneration.BuildDatabaseContext()).Returns(fakeDbContext);
+
+            var faceFeaturesHelper = new FaceFeaturesHelper(fakeDbContextGeneration);
+            
+            var expectedId = 0;
```

Comment:
Have some other number than 0, because its default value of int

Reply:
Strangely enough tests don't automatically generate unique ID while when working with real DB ID is generated by itself, so I just removed that check